src/app/components/AnalyticsConsent/index.tsx

```diff
@@ -111,9 +103,15 @@ export const ReopenAnalyticsConsentButton = () => {
   const context = useContext(AnalyticsContext)
   if (context === null) throw new Error('must be used within AnalyticsContext')
   return (
-    <StyledPrivacyButton size="small" color="inherit" onClick={() => context.reopenAnalyticsConsent()}>
-      {t('analyticsConsent.settings')}
-    </StyledPrivacyButton>
+    <Link asChild className="text-muted-foreground">
```

Comment:
textColor="inherit" instead className (this will also align hover state with other links in the footer)